### ISSUE TITLE
The axis is not clear

### ISSUE THREAD

**Issue by jasonhubs** [mohirio/jupyterlab-horizon-theme#1]
![image](https://user-images.githubusercontent.com/16680297/61017695-c2e2b400-a3c6-11e9-9571-ff1764711453.png)

**Comment by mohirio** [comment]
Hey, thanks for the notice.
The issue is fixed in 77876ed86954a9083efe2ee3f1c8f96e285cdf50 for ```JupyterLab >1.0.0```.

I also published a version on npm If you're using an older version of JupyterLab, please install with 
```bash
jupyter labextension install @mohirio/jupyterlab-horizon-theme@0.1.5
```

Although a better solution for this is to change the matplotlib style:
```python
plt.style.use('dark_background')
```

Or modify your [matplotlibrc](https://matplotlib.org/users/customizing.html) file to match the theme.

**Comment by jasonhubs** [comment]
@mohirio  ok! thanks for your reply!
